woocommerce/woocommerce
# Load same stylesheets in the Site Editor as in the frontend

### All Submissions:


### Changes proposed in this Pull Request:

Currently, in the Site Editor we are only loading two frontend styles: `/assets/css/woocommerce.css` and `/assets/css/woocommerce-blocktheme.css`. However, in some circumstances, there might be more CSS files or they might be replaced by others, for example, when using TT2 or TT3, which replace `/assets/css/woocommerce.css` with their own stylesheet. We would like to honor that in the editor so frontend and editor look as similar as possible.

This PR uses `WC_Frontend_Scripts::get_styles()` to get the list of styles which are loaded in the frontend to load them in the Site Editor.


### How to test the changes in this Pull Request:


1. You will need to install [WC Blocks 9.7.0 or higher](https://wordpress.org/plugins/woo-gutenberg-products-block/). This will enable the _Product Details_ block.
2. Install also [TT2](https://github.com/wordpress/twentytwentytwo) or [TT3](https://github.com/WordPress/twentytwentythree/) themes.
3. Go to Appearance > Editor and edit the Single Product template.
4. After the _WooCommerce Single Product Block_, add the _Product Details_ block.
5. Verify theme styles are applied properly:

Before | After
--- | ---
![imatge](https://user-images.githubusercontent.com/3616980/220364623-4a70c6b7-c1e9-4471-8be3-052e6cac968f.png) | ![imatge](https://user-images.githubusercontent.com/3616980/220364014-6a5ac461-56e1-42a1-9d39-30cabaa4b678.png)



### Other information:




### FOR PR REVIEWER ONLY:
